holoviews/streams.py

```diff
@@ -115,7 +61,7 @@ def trigger(cls, streams):
             stream.deactivate()
 
 
-    def __init__(self, preprocessors=[], source=None, subscribers=[],
+    def __init__(self, rename={}, source=None, subscribers=[],
```

Comment:
Did none of the docstrings mention preprocessors? Either way docstrings should now mention the rename argument.